app/assets/stylesheets/accounts.scss

```diff
@@ -34,6 +34,7 @@
     text-align: center;
     position: relative;
     z-index: 2;
+    text-shadow: 0 0 2px #000;
```

Comment:
Please use `$color8` instead of `#000` on all of these, and this is good to go. (color8 resolves to 000 but is configurable)